fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -253,7 +253,10 @@ public void execute() throws Exception {
     // Exception:
     //  IOException: talk with client failed.
     public void execute(TUniqueId queryId) throws Exception {
-
+        // origin value init
+        VariableMgr.setIsSingleSetVar(false);
+        VariableMgr.clearMapSessionOriginValue();
```

Comment:
VariableMgr is a thread safe class.
But `execute()` can be called by multi threads

Reply:
So  `isSingleSetVar` and `sessionOriginValue` should be set in `SessionVariables` instance.

Reply:
Got it and Fixed. sorry for my fault.